### ISSUE TITLE
Typedef arguments

### ISSUE THREAD

**Issue by victorgram** [matusnovak/doxybook2#9]
Hi @matusnovak 
We have a typedef that is a function pointer with arguments, in the xml produced by doxygen there is a property <argsstring> containing the arguments of the function pointer. However this property is not present in doxybook2 via the .tmpl files.
If possible, it would be greatly appreciated if this could be added.

**Comment by matusnovak** [comment]
Hi @victorgram 

This is the same issue as the https://github.com/matusnovak/doxybook2/issues/10 and the fix is now released as [releases/tag/v1.1.0](https://github.com/matusnovak/doxybook2/releases/tag/v1.1.0).

For future reference, I have added a sample code for function pointers in [example/src/Audio/AudioBuffer.hpp#L9-L23](https://github.com/matusnovak/doxybook2/blob/master/example/src/Audio/AudioBuffer.hpp#L9-L23) as an example and this is the output screenshot from [here](https://matusnovak.github.io/doxybook2/hugo-learn/namespaces/namespaceengine_1_1audio/#using-callback2):

![image](https://user-images.githubusercontent.com/10331278/80246738-32cbdd80-866d-11ea-8c1f-338a37450b7b.png)

**Note!**

Doxygen seems to be badly parsing the typedefs for function pointers. For example, the following:

```cpp
typedef int (*Callback)(AudioBuffer& buffer, size_t offset);
```

Gets converted to the following XML:

```xml
<memberdef kind="typedef" id="namespace_engine_1_1_audio_1a9527f056637d57771ee8025589fd596d" prot="public" static="no">
        <type>int(*</type>
        <definition>typedef int(* Engine::Audio::Callback) (AudioBuffer &amp;buffer, size_t offset)</definition>
        <argsstring>)(AudioBuffer &amp;buffer, size_t offset)</argsstring>
        <name>Callback</name>
</memberdef>
```

See the `<type>` ? I don't think that's correct and I can't do anything about it, so I am loading it and displaying it as it is defined in the XML, you get the following output:

![image](https://user-images.githubusercontent.com/10331278/80247039-b685ca00-866d-11ea-812d-0cf2e1d79c2b.png)

But, **Doxygen does parse it the type correctly for function pointers via the using keyword.** So, If you want to avoid that problem, use `using Name = ...;` instead.

If you look at the original html generated by the Doxygen, the issue is there as well and has been for a while. The only reason no one really notices is that because the `<type>` and `<argsstring>` from the Doxygen's XML is put into two html table cells that are right next to each other. Therefore, it appears "fine" when browsing the html version.

**Comment by matusnovak** [comment]
Please ignore the previous comment. I have fixed the issue with `int(*` being cut off.

Released as [releases/tag/v1.1.1](https://github.com/matusnovak/doxybook2/releases/tag/v1.1.1).

Should appear now correctly on list of types:

![image](https://user-images.githubusercontent.com/10331278/80249933-0d41d280-8673-11ea-8f55-eae940cca86c.png)

And an example detailed description for typedefs with parameters that were missing from [here](https://matusnovak.github.io/doxybook2/hugo-learn/namespaces/namespaceengine_1_1audio/):

![image](https://user-images.githubusercontent.com/10331278/80249991-2b0f3780-8673-11ea-91f5-64b7ceb23a4e.png)
